.github/workflows/label-pull-requests.yml

```diff
@@ -0,0 +1,153 @@
+name: Pull request labeler
+
+# FIXME: Consider adding the `issue_comment` event to change labels based on generic, non-review pull request comments.
+#        Consider the trade off of how spammy it can be (one CI run per comment) and how useful it would be to have.
+#        Consider alternatives to `issue_comment`.
+on:
+  pull_request:
+    types: [opened, reopened, converted_to_draft, ready_for_review, synchronize, edited, review_requested, closed]
+  pull_request_review:
+    types: [submitted, dismissed]
+
+jobs:
+  label_pull_request:
+    runs-on: ubuntu-22.04
+    if: always() && github.repository == 'SerenityOS/serenity'
+    
+    steps:
+      - name: Label pull request
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.BUGGIEBOT }}
+          script: |
+            const prCommunityApprovedLabel = "✅ pr-community-approved";
+            const prMaintainerApprovedButAwaitingCiLabel = "✅ pr-maintainer-approved-but-awaiting-ci";
+            const prNeedsReviewLabel = "👀 pr-needs-review";
+            const prWaitingForAuthorLabel = "⏳ pr-waiting-for-author";
+            const prIsBlockedLabel = "⛔️ pr-is-blocked";
+            const prUnclearLabel = "🤔 pr-unclear";
```

Comment:
can we make this an enum-like object?

```js
const Label = {
    PR_COMMUNITY_APPROVED: "✅ pr-community-approved",
    ...
};
```

and then

```js
Label.PR_COMMUNITY_APPROVED
```

seems a bit nicer than

```js
prCommunityApprovedLabel
```